src/network/send.ts

```diff
@@ -198,15 +201,28 @@ export async function sendMessage({
 
     // console.log("==> SENDER",sender)
     // console.log("==> OK SIGN AND SEND", opts);
-    try {
-      const r = await signAndSend(opts, sender, mqttTopic)
-      yes = r
-    } catch (e) {
-      sphinxLogger.error(`KEYSEND ERROR ${e}`)
-      no = e
+    if (!mqttTopic) {
+      try {
+        const r = await signAndSend(opts, sender, mqttTopic)
+        yes = r
+      } catch (e) {
+        sphinxLogger.error(`KEYSEND ERROR ${e}`)
+        no = e
+      }
+    } else {
+      let data = JSON.stringify(m)
+      const sig = await LND.signAscii(data, sender.publicKey)
+      data = data + sig
+      const batchItem = { mqtt_topic: mqttTopic, data: data }
+      batchContents.push(batchItem)
     }
     await sleep(10)
   })
+  console.log('SENDING TRIBE BATCH MESSAGE')
+  if (isTribeOwner) {
+    const tribeBatch = { chat_uuid: chat.uuid, batch_contents: batchContents }
+    await tribes.sendTribeBatchMessage(tribeBatch, sender.publicKey)
```

Comment:
@Evanfeenstra I'm trying to use this `sender.publicKey` value but when I'm calling the tribes server its giving me a `401` from what I've looked this seems to be the right `owner_pubkey` is there something I'm missing?

Reply:
Ya pretty sure it should be `owner.publicKey`... however those values should be the same anyway, since all messages in tribes have a `sender` that is the tribe admin (owner)

Any clues in tribes why the 401?

Reply:
In tribes I see the `tribe.OwnerPubKey` but for `pubKeyFromAuth` its showing up as blank when I log it which I'm not sure why

Reply:
Aha i see! In tribes, the new route `r.Post("/tribe/messageBundle", HandelTribeMessageBundleFromRelay)` needs to go within the code block that has the proper context: `r.Use(PubKeyContext)`